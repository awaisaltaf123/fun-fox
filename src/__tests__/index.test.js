jest.mock("../App", () => "div");

describe("ReactDOM render test", () => {
  let container;

  beforeEach(() => {
    container = document.createElement("div");
    container.setAttribute("id", "root");
    document.body.appendChild(container);

    require("../index");
  });

  afterEach(() => {
    document.body.removeChild(container);
    container = null;
  });

  it("renders the document body correctly", () => {
    expect(document.body).toMatchSnapshot();
  });
});
