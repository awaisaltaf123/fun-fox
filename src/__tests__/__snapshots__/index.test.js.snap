// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactDOM render test renders the document body correctly 1`] = `
<body>
  <div
    id="root"
  />
</body>
`;
